pennylane/templates/state_preparations/state_prep_mps.py

```diff
@@ -25,14 +27,17 @@ class MPSPrep(Operation):
 
     .. note::
 
-        Currently, this operator can only be used with ``qml.device(“lightning.tensor”)``.
+        This operator is natively supported on the ``lightning.tensor`` device, designed to run MPS structures efficiently. For other devices, implementing this operation uses a gate-based decomposition which requires auxiliary qubits (via ``work_wires``) to prepare the state vector represented by the MPS in a quantum circuit.
+
```

Comment:
Just curious, is there a reference to this implementation using a gate-based decomposition?

Reply:
Ah I see it is in the compute_decomposition function. not sure where it's preferred.

Reply:
I'll put it in both 😄